src/main/java/ToDo.java

```diff
@@ -0,0 +1,10 @@
+public class ToDo extends Task {
+    ToDo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[T]%s", super.toString());
```

Comment:
Hmm, I believe the .toString() is not necessary as the format specifier has already stated it is a string ("%s").